aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/SyncTimeRegistry.java

```diff
@@ -55,9 +56,26 @@
         });
     }
 
-    <T extends Model> Completable saveLastSyncTime(@NonNull Class<T> modelClazz, SyncTime syncTime) {
+    <T extends Model> Completable saveLastDeltaSyncTime(@NonNull Class<T> modelClazz,
+                                                   @Nullable SyncTime syncTime) {
```

Comment:
nit: alignment -- i'd either align arg, or wrap and +8 indent from method head

Reply:
Ugh...that's ugly. Just fixed it.